src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -120,6 +121,11 @@ class DocBaseViewer extends BaseViewer {
         this.loadTimeout = LOAD_TIMEOUT_MS;
 
         this.startPageNum = this.getStartPage(this.startAt);
+
+        const thumbnailsSidebarEl = document.createElement('div');
+        thumbnailsSidebarEl.classList.add(CLASS_BOX_PREVIEW_THUMBNAILS_CONTAINER, CLASS_HIDDEN);
+        this.containerEl.parentNode.insertBefore(thumbnailsSidebarEl, this.containerEl);
+        this.thumbnailsSidebarEl = thumbnailsSidebarEl;
```

Comment:
It looks like the existing convention is to assign the variable to the class instance and work with that rather than a local variable.

```
this.thumbnailsSidebarEl = document.createElement('div');
this.thumbnailsSidebarEl.classList.add(CLASS_BOX_PREVIEW_THUMBNAILS_CONTAINER, CLASS_HIDDEN);
// etc
```